app/uk/gov/hmrc/gform/commons/NumberFormatUtil.scala

```diff
@@ -20,13 +20,28 @@ import java.text.NumberFormat
 import java.util.Locale
 import java.math.RoundingMode
 
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.{ RoundingMode => GformRoundingMode }
+
 object NumberFormatUtil {
   val defaultFormat = NumberFormat.getInstance(Locale.UK)
-  def defaultFormat(i: Int) = {
+  def defaultFormat(i: Int, roundingMode: GformRoundingMode) =
+    RoundingModeUtil.RoundingFormat(i, roundingMode)
+  val currencyFormat = NumberFormat.getCurrencyInstance(Locale.UK)
+}
+
+object RoundingModeUtil {
+  def RoundingFormat(digits: Int, roundingMode: GformRoundingMode) = {
```

Comment:
What about instead of two parameters, we pass `MaxDigitsAndRoundingMode` instead? That would made a upstream method definition much cleaner.